src/styles/utils/normalize.scss

```
@@ -0,0 +1,15 @@
+* {
+  box-sizing: border-box;
+}
+
+body {
+  font-family: "Roboto", sans-serif;
+  font-style: normal;
+}
```

```suggestion
html {
  font-family: "Roboto", sans-serif;
}
```
always set the font to the HTML and remove `font-style: normal` - its default